tests/unit/amo/sagas/test_collections.js

```diff
@@ -327,164 +328,248 @@ describe(__filename, () => {
     });
   });
 
-  describe('updateCollection', () => {
+  describe('modifyCollection', () => {
+    const formOverlayId = 'some-form-overlay';
+
+    const _createCollection = (params = {}) => {
+      sagaTester.dispatch(createCollection({
+        errorHandlerId: errorHandler.id,
+        formOverlayId,
+        name: 'some-collection-name',
+        slug,
+        user,
+        ...params,
+      }));
+    };
+
     const _updateCollection = (params = {}) => {
       sagaTester.dispatch(updateCollection({
         errorHandlerId: errorHandler.id,
-        formOverlayId: 'some-form-overlay',
+        formOverlayId,
         collectionSlug: 'some-collection',
-        user: 'some-user',
+        user,
         ...params,
       }));
     };
 
-    it('sends a patch to the collection API', async () => {
-      const params = {
-        collectionSlug: 'a-collection',
-        description: { 'en-US': 'New collection description' },
-        formOverlayId: 'some-form-overlay',
-        name: { 'en-US': 'New collection name' },
-        slug: 'new-slug',
-        user: 543,
-      };
-      const state = sagaTester.getState();
+    describe('common logic', () => {
+      // The common logic can be tested either by dispatching updateCollection
+      // or createCollection, so we just use _updateCollection.
```

Comment:
I like this approach to the tests given the circumstances. The comment helps explain things too.

Reply:
I know fast tests are good and we don't want to go overboard, but testing each action feels a bit safer for future refactors for me. I'm okay with this though, it's cool. 🤷‍♂️